.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+php:
+  - 7.1
+
+cache:
+  directories:
+    - ~/.composer/cache
+    - $HOME/.npm
+
+env:
+  - ELASTIC_SEARCH_INDEX_PREFIX=""
+
+before_install:
+  - echo "extension = redis.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
+  - phpenv config-rm xdebug.ini || return 0
+
+install:
+  - composer global require hirak/prestissimo
```

Comment:
Wonder if this shouldnt be part of the before_install section. What do you think? Because this is not really an install thing in my opinion, more like setting a helper for installation

Reply:
I moved that